### ISSUE TITLE
Rows per page uses node ID instead of counting items causing items to be hidden

### ISSUE THREAD

**Issue by KennethLavrsen** [OpenZWave/Zwave2Mqtt#37]
I have been fighting to get a simple setup with an Aeotec stick and two sirens to work.
I moved them over from a Home Assistant installation and it used to be a larger installation

I tried to import the config and it was reported successful but I assumed it did not work because I could not see my 2 sirens.

So I removed them and added them and they are in the config. But no sign of them in the user interface - in the table of nodes. Until I by accident touch the header and the table sorted descending. Then I tried to click to see ALL instead of rows per page 10. Then all 3 nodes are visible

So the table of nodes and the feature of showing 10 at a time uses the node number. That is wrong. An Aeotec stick will keep on assigning higher numbers even when you remove older nodes. In my case my first node was in the upper 20s. And after having added and removed they became 30 and 31. I then tried to factory reset the stick and start all over. Then the nodes are 1,2,3 and shows normally. You should change the table showing feature so it shows the 10 first, Then the next 10, and next 10. Not by node ID buy by counting. People will have holdes in their node intervals so the tables will be strange with less than 10 in  them otherwise and worst case people like me will not see any nodes

**Comment by robertsLando** [comment]
There is a flag in the table to show/hide the hidden nodes, if you remove a node the next you add will get the next free nodeid, if you want a node to take a nodeid you need to use the function `replaceFailedNode`

**Comment by KennethLavrsen** [comment]
That is not the problem. In my case I had removed many nodes so my Aeotec was number 1 and the first node was 28. I was fighting for an hour adding and removing nodes and could not understand that they did not show up in the table. I did press hidden nodes and saw 1 Aeotec and just lines saying deleted nodes. The user interface does not in any way hint that your nodes are there on page 2 or 3. It put me in trouble and I am sure others than I will fall into that trap. The best would be to page the table so that if you have 3 nodes they are on page 1 no matter what numbers they have. That is what you expect of all software
